INSTALL.rst

```diff
@@ -304,13 +304,52 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+Running the network server:
+---------------------------
 
-  haas serve_networks &
 
-as the HaaS user. To make this happen on boot, add the following to ``/etc/rc.local``::
+A systemd script for running the network server is available in the 'scripts' directory.
+Name of the script is: haas_network.service
+
+Centos:
+------
+
+Centos uses systemd to controll all its processes. 
+
+Place the file haas_network.service under:
+``/usr/lib/systemd/system/``
```

Comment:
I think I agree with Sahil on this - on a random ubuntu 15.10 system, /etc/systemd just has a bunch of symlinks to /lib/systemd/...:

```
$ find /etc/systemd -type f
/etc/systemd/logind.conf
/etc/systemd/timesyncd.conf
/etc/systemd/user.conf
/etc/systemd/bootchart.conf
/etc/systemd/system.conf
/etc/systemd/journald.conf
/etc/systemd/resolved.conf
```